holidays.py

```diff
@@ -2194,6 +2196,68 @@ class NL(Netherlands):
     pass
 
 
+class Egypt(HolidayBase):
+    SUNDAY = 1
```

Comment:
this variable is not used, what's its purpose?

Reply:
It was copied and modified from the `Netherlands` class, but indeed the variable is not used.